reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+The authors then propose their own view: "the objective of preregistration is not the maximization of theoretical risk but rather the minimization of uncertainty about the theoretical risk."
+I think the authors are in the right direction, but miss some things.
+What the authors get right is that preregistration is about communicating something about the test to others.
+
+> This is why we deliberately chose to use the term "uncertainty" and not "control".
+> Reducing uncertainty directly invokes the concept of communication.
+> Control also seems to imply that it is a property of the procedure, as it seems curious to suggest that control differs between the reader and the original researcher (we argue that the uncertainty is lower for the latter as they have all the information).
+
+And the authors are correct that preregistration can remove uncertainty if all readers of a preregistration 1) agree on the analyses plan, and 2) there are no deviations of the preregistration.
+
+> We make no such claim.
+> We explicitly address that readers may differ considerably in how highly they judge the theoretical risk and their confirmation function.
+> *One feature of the Bayesian framework is the strong emphasis on subjective yet rational judgement.*
+> *Therefore, we assume that researchers will differ significantly in how they value evidence but that by making assumptions about the general process, we can make general statements that apply to all these subjective evaluations.*
+> They may, therefore, very well disagree on the analysis.
+> We also discuss at length under which circumstances a post hoc change to the preregistration is justified (if the increase in confirmation does offset the negative effects of uncertainty).
+
+However, the authors already say 2 is not always true, as people often need to deviate.
+And 1 is also not true.
+This is important, because if 1 and 2 are true, then we can not minimize uncertainty about theoretical risk (as the authors argue the goal of preregistration is), because this uncertainty is based on a subjective evaluation.
+If you deviate from an analysis plan because you think after looking at the data that you should have excluded some outliers, but this was not specified in the preregistration, as a reader I am left with an evaluation of whether 1) this analysis is a *better* test, and it reduces the possibility of incorrectly claiming there was an effect, because these outliers might make a null effect significant, while they should have been removed, or 2) this analysis is a *worse* test, and the reader believes the choice for this test without the outliers is driven by the nicer results the researcher observed.
+The preregistration does not minimize uncertainty about the Type 1 error rate.
+For some readers, the deviation increased the Type 1 error rate, and for others it did not.
+
+> This aligns with our reasoning: Any change should increase uncertainty about the theoretical risk.
```

Comment:
I like Laken's example. Couldn't we integrate it into the manuscript? It's probably a common type of deviation and it's easy to understand that there are differences in subjective judgements between readers and authors.